src/main/java/org/gridsuite/directory/server/services/UserAdminService.java

```diff
@@ -40,17 +37,6 @@ public UserAdminService(RestTemplate restTemplate, RemoteServicesProperties remo
         this.restTemplate = restTemplate;
     }
 
-    public boolean isUserAdmin(String sub) {
-        String path = UriComponentsBuilder.fromPath(DELIMITER + USER_ADMIN_API_VERSION + IS_USER_ADMIN_URI)
-                .buildAndExpand(sub).toUriString();
-        try {
-            var responseEntity = restTemplate.exchange(userAdminServerBaseUri + path, HttpMethod.HEAD, HttpEntity.EMPTY, Void.class);
-            return responseEntity.getStatusCode().is2xxSuccessful();
-        } catch (HttpStatusCodeException e) {
-            return false;
-        }
-    }
```

Comment:
Just change the implementation of this method by checking if ADMIN_EXPLORE is present, then it will change almost nothing in the code

Reply:
ok then you can make it simpler for RoleService